src/Std/Data/DTreeMap/Raw/WF.lean

```diff
@@ -161,4 +161,8 @@ theorem union [TransCmp cmp] {t₁ t₂ : Raw α β cmp} (h₁ : t₁.WF) (h₂
     (t₁.union t₂).WF :=
   ⟨Impl.WF.union! h₁.out h₂.out⟩
 
+theorem inter [TransCmp cmp] {t₁ t₂ : Raw α β cmp} (h₁ : t₁.WF) :
+    (t₁.inter t₂).WF :=
```

Comment:
This and all similar lemmas should be stated in terms of `∩` rather than `inter`.